changelog-2025.06.12-3.patch

```diff
@@ -1,11 +1,21 @@
 diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
-index 989df8d0ed..ff08ba2564 100644
+index 989df8d0ed..8a37dec1fd 100644
 --- a/packaging/app.organicmaps.desktop.metainfo.xml
 +++ b/packaging/app.organicmaps.desktop.metainfo.xml
-@@ -115,6 +115,22 @@
+@@ -115,6 +115,32 @@
    </screenshots>
  
    <releases>
++   <release version="2025.06.12-3" date="2025-06-12">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>OpenStreetMap data as of June 8</li>
```

Comment:
Thanks! Here is the full list of desktop changes:

• New OpenStreetMap data as of June 8, 2025
• Less annoying icons for car charging stations
• Display plant nurseries, crash/traffic barriers, radio/tv/music studios, firepits, ladders, cranes, and love hotels
• Qingdao metro icons
• Updated sewing/tailor search synonyms  
• Fixed category search for Lithuanian and other languages
• Added missing outdoors style to the bundle
• Fix "About" dialog window flags and content

Reply:
OK the open PRs will be:
https://github.com/organicmaps/organicmaps/pull/10710
https://github.com/flathub/app.organicmaps.desktop/pull/101